gax-java/gax/src/main/java/com/google/api/gax/tracing/ApiTracer.java

```diff
@@ -121,7 +123,10 @@ default Scope inScope() {
    * @param error the transient error that caused the attempt to fail.
    * @param delay the amount of time to wait before the next attempt will start.
    */
-  default void attemptFailedDuration(Throwable error, java.time.Duration delay) {};
+  default void attemptFailedDuration(Throwable error, java.time.Duration delay) {
+    // defaults to do the same as attemptFailed(Throwable, org.threeten.bp.Duration)
```

Comment:
nit: Can you add a blurb to reflect the `why`? 

i.e. Customers may have older/legacy code that directly implements ApiTracer's `attemptFailed()` method and their overriden logic should be invoked in gax.

Reply:
Good suggestion, thank you. I added a similar blurb in the body comment.